settings

```diff
@@ -20,18 +20,20 @@ load_settings() {
   # Load settings per release
   # Expected: FULLGPGKEY
```

Comment:
Technically this also isn't always required. Otherwise we should add this to the "template".